server/routes/todoListRouter.js

```diff
@@ -1,24 +1,102 @@
 const todoListRouter = require('express').Router();
+const query = require('../controllers/pg.js');
 
 
+todoListRouter.get('/info', (req, res) => {
+  const email = req.params.email || '1@qq.com';
+  try {
+    query.getInfo(email, async(err, response) => {
+      if (err) {
+        res.status(400).send('get question error');
+      } else {
+        const result = { results: response.rows };
+        res.status(200).send(result);
+      }
+    });
+  } catch(error) {
+    console.error(error);
+    res.send({data: error});
+  }
+});
 
-// Routes needed:
-// GET ':userEmail' -> For all data
-// POST ':userEmail' -> Adding or Upserting a "todoList item"
-// PATCH ':userEmail' -> For updating the data -> ex. Moving around item in Calendar / Lengthening item in Calendar / Clicking on "Done" in Modal for Calendar/TodoList
-// DELETE ':userEmail' -> For deleting the data -> Clicking on "Delete" button in Modal
+todoListRouter.post('/category', async (req, res) => {
+  var calendarId = req.body.calendarId || 2;
+  var category = req.body.category || 'do some homework';
+  query.postCategory(calendarId, category, async (err, response) => {
+    if (err) {
+      res.status(400).send('post category error');
+    } else {
+      res.status(201).send('category posted');
+    }
+  });
+});
 
-// The above list is also in Home.jsx, so also make the appropriate changes there, as well.
+todoListRouter.post('/item', async (req, res) => {
+  var title = req.body.title || 'opoipoi with E';
+  var description =  req.body.description || 'oipoipiwith E';
+  var duration =  req.body.duration || 3600;
+  var start =  req.body.start || '2022-04-20T10:30:20';
```

Comment:
These defaults seem fine for our next demo but not great for a production server. I feel like current day/time would be ideal for production.